### ISSUE TITLE
Outreach

### ISSUE THREAD

**Issue by KadeMorton** [aletheia-foundation/aletheia-admin#46]
Go through the open science twitter feed and others and reach out to people passionate about Open Access.

**Comment by chartgerink** [comment]
Suggestions

- https://twitter.com/MikeTaylor
- https://twitter.com/petersuber
- https://twitter.com/ernestopriego
- https://twitter.com/R2RC
- https://twitter.com/p_binfield
- https://twitter.com/hjoseph
- https://twitter.com/jeroenbosman
- https://twitter.com/CameronNeylon
- https://twitter.com/brembs
- https://twitter.com/Impactstory
- https://twitter.com/OpenAccessEC
- https://twitter.com/openatvt
- https://twitter.com/l_matthia
- https://twitter.com/protohedgehog

Can go on for quite a bit but this should be a good starting point :open_hands:

**Comment by KadeMorton** [comment]
Thanks so much @chartgerink I'll get to this!

**Comment by lmatthia** [comment]
If we're gonna do this on a larger scale, it might be more efficient to reach out to people with a mail merge.

**Comment by KadeMorton** [comment]
I agree that is certainly more scalable. My only fear is people hate unpersonalised mail, and it might seem like they got added to a mailing list without their permission. What are your thoughts on this, @lmatthia ?

**Comment by KadeMorton** [comment]
https://www.creativechain.org/project/ - look over the outlets that gave them press and reach out to them.

**Comment by KadeMorton** [comment]
People should feel free to add more and draft up emails. Please speak to a maintainer before sending!
